src/WebdriverClassicDriver.php

```diff
@@ -751,8 +766,8 @@ public function getBrowserName(): string
      */
     public function getWebDriverSessionId(): ?string
     {
-        return $this->isStarted()
-            ? $this->getWebDriver()->getSessionID()
+        return $this->isStarted() && method_exists($this->getWebDriver(), 'getSessionId')
+            ? $this->getAsString($this->getWebDriver()->getSessionID(), 'Session ID')
```

Comment:
You're checking, that the `getSessionId` method exists, but calling `getSessionID`. Please use the same case (based on method declaration).